app/controllers/foreman_bootdisk/disks_controller.rb

```diff
@@ -40,6 +40,23 @@ def full_host
       end
     end
 
+    def subnet
+      begin
+        subnet = @disk.try(:subnet)
```

Comment:
Magic in the base controller, named after the base resource which is a "disk" in this case, except the class is actually a Host.